tests/bluetooth/tester/qemu.conf

```diff
@@ -1,49 +0,0 @@
-CONFIG_TEST=y
```

Comment:
Do we really want to remove qemu support or is it because prj.conf has the same configuration?

Reply:
Correct, now `prj.conf` is concatenated with `qemu_cortex_m3.conf` automagically by the build system.

Reply:
The qemu support stays exactly the same as before.